src/game/models/asteroid.rs

```diff
@@ -6,7 +6,7 @@ use piston_window::{Context, polygon, Size, Transformed, UpdateArgs};
 use rand;
 
 use game::color;
-use game::models::{Collidable, Drawable, PI_MULT_2, Positioned, Updateable};
+use game::models::{Collidable, Drawable, PI, PI_MULT_2, Positioned, Updateable};
```

Comment:
We had to define `PI_MULT_2` because `std` doesn't expose it, and [not every is onboard with "tau"](https://github.com/rust-lang/rfcs/issues/1486).